src/daemon/database/schema.rs

```diff
@@ -89,6 +89,7 @@ CREATE TABLE spend_inputs (
     id INTEGER PRIMARY KEY NOT NULL,
     unvault_id INTEGER NOT NULL,
     spend_id INTEGER NOT NULL,
+    has_priority BOOLEAN CHECK (has_priority IN (0,1, NULL)),
```

Comment:
+1 for the check :+1:

What if we make this just a boolean and `false` by default? A future `prioritisetransaction` RPC command could just set this to `true` and `updatespendtx` with the optional priority parameter would also updated it to `true`.